src/test/resources/features/highlighting/inspections/correct_using_methods.feature

```diff
@@ -1,24 +1,42 @@
 Feature: Inspection for methods from snakemake library
 
-  Scenario Outline: Unnecessary methods in rule/checkpoint section
+  Scenario Outline: Incorrect using ancient/protected/directory methods
     Given a snakemake project
     Given I open a file "foo.smk" with text
     """
     <rule_like> NAME:
-      <section>: <method>
+       <section>: <method>
     """
     And SmkCorrectUsingMethodsInspection inspection is enabled
     Then I expect inspection warning on <<method>> in <<section>: <method>> with message
     """
-    '<method>' doesn''t make sense in '<section>'
+    '<method>' doesn't make sense in '<section>'
```

Comment:
Please use a more formal error message, e.g. `... isn't supported in ...`

Reply:
Done